tests/System.Buffers.Primitives.Tests/CopyTo.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Xunit;
+
+namespace System.Buffers.Tests
+{
+    public class CopyTo
+    {
+        [Fact]
+        public static void CopyToSpan()
+        {
+            int[] src = { 1, 2, 3 };
+            int[] dst = { 99, 100, 101 };
+
+            var srcBuffer = new Buffer<int>(src);
```

Comment:
Since you added the implicit cast, you should try to exercise it here and get extra coverage.